### ISSUE TITLE
Email with draft hidden in bundle

### ISSUE THREAD

**Issue by nilnocb** [leggett/simplify#656]
When viewed at root of inbox, can see thread with labels. When click on email with thread that has a draft in it (labels include inbox and custom bundle label and isread = true), the thread with draft is hidden. Needed to go to drafts or custom label only (i.e label="xxx") to view thread and edit the draft.
---
Simplify v.3.0.4 - Configuration: aAX addOnsOpen appMenuOpen bundleInbox ca3 caI caL cfC cfI cfO cfQ cfR cfU composeActions composeFormat darkTheme dateGroup fontMsg hideListCount hideMsgCount hideSelectRefresh hideSignatures hideTabIcons inMsg invertAddons invertCompose izBgDefault listWidthLg lowDensity minimizeChat msgLabelsRight msgWidthLg nPane newUI reverseMsgs rightStar showAvatars simplify simplify-canary simplifyMenuOpen supExtWarn - System: Mozilla/5.0 (X11 Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36 - Window: 2256 x 1460 (175pct zoom) - Language: en

**Comment by leggett** [comment]
Ok, this is a super fun bug that gets deep deep into how Gmail actually works. My upcoming update to inline expanding bundles will mostly fix it but there isn't really anything I can do for when you go to a search for label:x in:inbox. 

I'll try to explain...

While you see threads in Gmail, there are no threads in the backend. There are emails with IDs that pull them together into a thread. So, labels are applied to emails, not threads.

Simplify only sees what is on the page. In the inbox, that means I see threads, not emails. I see there is a thread with two emails and Gmail shows the label "xxx" on that thread. But that label isn't on all the emails in that thread. That is the issue. Here is why:

You've got an email that you probably manually applied a label to and then archived (removed the inbox label). Then a new reply came in (automatically gets the inbox label but not the xxx label). The first email in that thread has your label but not the inbox label. The second email has the inbox label but not your label. That's why searching for label:xxx in:inbox doesn't show the thread (there isn't an email in that thread with both of those labels). Search for just label:xxx and you see it. 

I wrote up a massive proposal when I worked on Gmail to fix all this and I couldn't convince the backend team the issue was worth the trouble. 

Bottom line: bundle searches only really work well with labels applied via filter as it re-applies the label to new messages as they come in OR it works for those that don't keep a clean inbox (remember, archiving meant the first message no longer had both the inbox and your label). When I add support for inline expanding bundles, it will fix this in the inbox but not for the searches.
